code/game/objects/items/holy_weapons.dm

```diff
@@ -5,7 +5,7 @@
 	desc = "It has the unyielding gaze of a god eternally forgotten."
 	icon_state = "clockwork_helmet"
 	inhand_icon_state = "clockwork_helmet_inhand"
-	armor = list(MELEE = 50, BULLET = 10, LASER = 10, ENERGY = 10, BOMB = 0, BIO = 0, FIRE = 80, ACID = 80)
+	armor = ARMOR_MEDIUM_BLUNT_HEAD
```

Comment:
I'd suggest giving the chappie armors the riot suit armor bonuses, as they're supposed to be equivalent to that. Apparently. Not like that can't be reconsidered of course.